examples/server/server.cpp

```diff
@@ -26,6 +26,28 @@ struct server_params {
     int32_t write_timeout = 600;
 };
 
+// completion string output with probabilities 
+struct completion_string_output {
+    struct token_prob {
+        std::string tok_str;
+        float prob;
+    };
+
+    std::vector<token_prob> probs;
+    std::string tok_str;
+};
```

Comment:
Is it necessary to copy the strings? Maybe `completion_token_output` is all that is needed, it is very easy to go from token to string.

Reply:
Yes, I removed it in the newest commit.